Resources/public/templates/ding2/slides/event/event-edit.html

```diff
@@ -0,0 +1,53 @@
+<div
+  class="event"
+  data-ng-style="{'font-size': theStyle.fontsize, 'width': theStyle.width, 'height': theStyle.height}"
+>
+  <div class="event--header-wrapper">
+    <div
+      class="event--header has-placeholder"
+      data-ng-style="{'color': ikSlide.options.headercolor}"
+    >
+      <div>{{ikSlide.options.header1}}</div>
+      <div>{{ikSlide.options.header2}}</div>
+      <div>{{ikSlide.options.header3}}</div>
+    </div>
+  </div>
+
+  <div class="event--info-wrapper">
+    <div
+      class="event--image"
+      data-ng-style="{'background-image': 'url(' + ikSlide.currentImage + ')'}"
+    ></div>
+    <div
+      class="event--info-header"
+      spellcheck="false"
+      data-ng-style="{'border-color': ikSlide.options.headercolor}"
+    >{{ ikSlide.options.date.text ? ikSlide.options.date.text : ikSlide.options.date.to
+      && ikSlide.options.date.to ? 'Dato: ' + ikSlide.options.date.from + ' - ' +
```

Comment:
How about handling this in a js function in the controller? The slideshow slide adds a functions to ng's scope and calls it from its template - maybe you could do the same to get rid of the nested ternary?

https://github.com/os2display/default-template-bundle/blob/master/Resources/public/apps/toolsModule/slideshow-order-editor.js#L24

https://github.com/os2display/default-template-bundle/blob/master/Resources/public/apps/toolsModule/slideshow-order-editor.html#L25

Reply:
:+1:

Reply:
The ternary has been vanquished from the kingdom.